tests/testdata/multicluster/envoy_local_v2.yaml

```diff
@@ -7,11 +7,15 @@ dynamic_resources:
   lds_config:
     api_config_source:
       api_type: GRPC
-      cluster_names: [xds_cluster]
+      grpc_services:
+        envoy_grpc":
```

Comment:
random `"`

Reply:
Good catch! Unfortunately, that wasn't the issue.